ubuntu-20.04/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:20.04
+
+# Install APT packages
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
The recommendation is to use ARG for variables which are only needed during the build of the image. See https://serverfault.com/questions/618994/when-building-from-dockerfile-debian-ubuntu-package-install-debconf-noninteract#797318

Reply:
Changed to ARG.